Wikipedia/Code/PreviewAndSaveViewController.m

```diff
@@ -122,13 +122,21 @@ - (BOOL)prefersStatusBarHidden {
 }
 
 - (void)wmf_showAlertForTappedAnchorHref:(NSString *)href {
+    
     UIAlertController *alertController =
-        [UIAlertController alertControllerWithTitle:href
-                                            message:nil
+        [UIAlertController alertControllerWithTitle:@"Your changes have not been saved yet"
```

Comment:
I was able to do this without issues. Will send in a new commit once I'm done with the other comment too.